pkg/controllers/machine/disruption/emptiness.go

```diff
@@ -0,0 +1,93 @@
+/*
```

Comment:
They aren't doing the same things. This is an intermediary state, but, right now, we have the node reconciler that is propagating the node emptiness TTL into the annotations on the node. The Machine disruption controller is looking for this annotation and then propagating the `Empty` condition on the Machine only if we are currently past that time plus the TTL.

The goal is to move this TTL over the machine disruption flow as well so that everything is happening over there in one spot.